packages/arb-bridge-eth/contracts/validator/GasRefunder.sol

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier: Apache-2.0
+
+/*
+ * Copyright 2021, Offchain Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+pragma solidity ^0.8.7;
+
+import "./IGasRefunder.sol";
+
+import "@openzeppelin/contracts-0.8/access/Ownable.sol";
```

Comment:
We want to use `OwnableUpgradeable` from `https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/v4.3.1/contracts/access/OwnableUpgradeable.sol`.

Reply:
If we do so, we should set the constructor of this contract as an initializer and surface an initializer here that calls this:
https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/blob/v4.3.1/contracts/access/OwnableUpgradeable.sol#L28

Reply:
no proxy will be used

Reply:
no proxy will be used